### ISSUE TITLE
tesla just got updated today tp 1.10.2-1.2.1.40

### ISSUE THREAD

**Issue by kreezxil** [Thutmose/Thut#7]
And your tesla hander appears to be having a non fatal server error with it, this is from the console of my dedicated server. Which is running Forge build 2018

```
[00:01:02] [Server thread/ERROR] [FML]: Exception caught during firing event net.minecraftforge.event.AttachCapabilitiesEvent$Entity@28d7148e:
java.lang.NoClassDefFoundError: thut/tech/common/tesla/TeslaHandler$2Provider
        at thut.tech.common.tesla.TeslaHandler.onEntityCapabilityAttach(TeslaHandler.java:123) ~[TeslaHandler.class:?]
        at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_436_TeslaHandler_onEntityCapabilityAttach_Entity.invoke(.dynamic) ~[?:?]
        at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:72) ~[ASMEventHandler.class:?]
        at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:168) [EventBus.class:?]
        at net.minecraftforge.event.ForgeEventFactory.gatherCapabilities(ForgeEventFactory.java:523) [ForgeEventFactory.class:?]
        at net.minecraftforge.event.ForgeEventFactory.gatherCapabilities(ForgeEventFactory.java:513) [ForgeEventFactory.class:?]
        at net.minecraft.entity.Entity.<init>(Entity.java:205) [rw.class:?]
        at net.minecraft.entity.EntityLivingBase.<init>(EntityLivingBase.java:155) [sf.class:?]
        at thut.tech.common.entity.EntityLift.<init>(EntityLift.java:275) [EntityLift.class:?]
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) [?:1.8.0_101]
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) [?:1.8.0_101]
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) [?:1.8.0_101]
        at java.lang.reflect.Constructor.newInstance(Constructor.java:423) [?:1.8.0_101]
        at mods.flammpfeil.slashblade.core.ConfigEntityListManager.onWorldTickEvent(ConfigEntityListManager.java:75) [ConfigEntityListManager.class:?]
        at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_365_ConfigEntityListManager_onWorldTickEvent_WorldTickEvent.invoke(.dynamic) [?:?]
        at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:72) [ASMEventHandler.class:?]
        at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:168) [EventBus.class:?]
        at net.minecraftforge.fml.common.FMLCommonHandler.onPreWorldTick(FMLCommonHandler.java:282) [FMLCommonHandler.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:694) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:387) [ld.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:613) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:471) [MinecraftServer.class:?]
        at java.lang.Thread.run(Thread.java:745) [?:1.8.0_101]
Caused by: java.lang.ClassNotFoundException: thut.tech.common.tesla.TeslaHandler$2Provider
        at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191) ~[launchwrapper-1.12.jar:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424) ~[?:1.8.0_101]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357) ~[?:1.8.0_101]
        ... 23 more
Caused by: java.lang.NoClassDefFoundError: net/darkhax/tesla/api/BaseTeslaContainer
        at java.lang.ClassLoader.defineClass1(Native Method) ~[?:1.8.0_101]
        at java.lang.ClassLoader.defineClass(ClassLoader.java:763) ~[?:1.8.0_101]
        at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142) ~[?:1.8.0_101]
        at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:182) ~[launchwrapper-1.12.jar:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424) ~[?:1.8.0_101]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357) ~[?:1.8.0_101]
        ... 23 more
Caused by: java.lang.ClassNotFoundException: net.darkhax.tesla.api.BaseTeslaContainer
        at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:101) ~[launchwrapper-1.12.jar:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424) ~[?:1.8.0_101]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357) ~[?:1.8.0_101]
        at java.lang.ClassLoader.defineClass1(Native Method) ~[?:1.8.0_101]
        at java.lang.ClassLoader.defineClass(ClassLoader.java:763) ~[?:1.8.0_101]
        at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142) ~[?:1.8.0_101]
        at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:182) ~[launchwrapper-1.12.jar:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424) ~[?:1.8.0_101]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357) ~[?:1.8.0_101]
        ... 23 more
[00:01:02] [Server thread/ERROR] [FML]: Index: 8 Listeners:
[00:01:02] [Server thread/ERROR] [FML]: 0: NORMAL
[00:01:02] [Server thread/ERROR] [FML]: 1: ASM: org.cyclops.cyclopscore.modcompat.capabilities.CapabilityConstructorRegistry@764610e6 onEntityLoad(Lnet/minecraftforge/event/AttachCapabilitiesEvent$Entity;)V
[00:01:02] [Server thread/ERROR] [FML]: 2: ASM: org.cyclops.cyclopscore.modcompat.capabilities.CapabilityConstructorRegistry@35af165 onEntityLoad(Lnet/minecraftforge/event/AttachCapabilitiesEvent$Entity;)V
[00:01:02] [Server thread/ERROR] [FML]: 3: ASM: org.cyclops.cyclopscore.modcompat.capabilities.CapabilityConstructorRegistry@6bae08a6 onEntityLoad(Lnet/minecraftforge/event/AttachCapabilitiesEvent$Entity;)V
[00:01:02] [Server thread/ERROR] [FML]: 4: ASM: org.cyclops.cyclopscore.modcompat.capabilities.CapabilityConstructorRegistry@496ce3c2 onEntityLoad(Lnet/minecraftforge/event/AttachCapabilitiesEvent$Entity;)V
[00:01:02] [Server thread/ERROR] [FML]: 5: ASM: mcjty.lib.McJtyLib$EventHandler@4f4281a9 onEntityConstructing(Lnet/minecraftforge/event/AttachCapabilitiesEvent$Entity;)V
[00:01:02] [Server thread/ERROR] [FML]: 6: ASM: airbreather.mods.pigmanure.AttachCapabilitiesEventHandler@3326a5fe invoke(Lnet/minecraftforge/event/AttachCapabilitiesEvent$Entity;)V
[00:01:02] [Server thread/ERROR] [FML]: 7: ASM: slimeknights.tconstruct.gadgets.item.ItemPiggybackPack@978cd0c attachCapability(Lnet/minecraftforge/event/AttachCapabilitiesEvent$Entity;)V
[00:01:02] [Server thread/ERROR] [FML]: 8: ASM: thut.tech.common.tesla.TeslaHandler@5ff3356a onEntityCapabilityAttach(Lnet/minecraftforge/event/AttachCapabilitiesEvent$Entity;)V
[00:01:02] [Server thread/ERROR] [FML]: 9: ASM: mcjty.rftools.ForgeEventHandlers@27b1efc3 onEntityConstructing(Lnet/minecraftforge/event/AttachCapabilitiesEvent$Entity;)V
[00:01:02] [Server thread/ERROR] [FML]: 10: ASM: pl.asie.charset.tweaks.minecart.TweakDyeableMinecarts@1891669b onAttachCapabilities(Lnet/minecraftforge/event/AttachCapabilitiesEvent;)V

```

**Comment by Thutmose** [comment]
Hmm, I will get to fixing this asap.

**Comment by Thutmose** [comment]
This should be fixed in version 3.0.6 which I just uploaded to curse, it should be available shortly.
